Library/Formula/node.rb

```diff
@@ -25,6 +25,7 @@ class Node < Formula
   option "without-completion", "npm bash completion will not be installed"
 
   depends_on :python => :build
+  depends_on :xcode => :build
```

Comment:
Thanks for checking. Weird that the CLT is the only difference between systems having the problem, and creating the bottled ` npm install ` fixes the issue, but that Xcode shouldn't be the issue any more. This is going to bug me, heh. At least bottling the ` make install ` seems to work for the people affected, even if we can't quite narrow down the root cause.